book_source/03_topical_pages/03_pecan_xml.Rmd

```diff
@@ -835,13 +839,17 @@ The following tags can be used for state data assimilation. More detailed inform
 ```
 
 * **process.variance** : [optional] TRUE/FLASE flag for if process variance should be estimated (TRUE) or not (FALSE). If TRUE, a generalized ensemble filter will be used. If FALSE, an ensemble Kalman filter will be used. Default is FALSE.
+* **aqq.Init** : [optional] The initial value of aqq used for estimate the Q distribution, the default value is 1.
+* **bqq.Init** : [optional] The initial value of bqq used for estimate the Q distribution, the default value is 1.
```

Comment:
1. Fixed.
2. It is the initial values for the aqq and bqq of the Gamma distribution because it is how we initialized the Gamma distribution historically.
3. The `pre_enkf_params` implies the pre-existing time series of process errors in the current SDA runs to ignore the impacts of the differences between process errors. It's now documented in the roxygen structure of the `sda.enkf.multisite` script.
4. The `aqq. init` and `bqq. init` right now only works on the `VECTOR` q type. We didn't account for their variabilities across sites or variables, meaning we initialized the `aqq` and `bqq` given single values. I documented them inside the `03_pecan_xml.Rmd` documentation.
5. The unit normalization has yet to come in so far.